src/deploy-events.js

```diff
@@ -9,10 +9,11 @@ module.exports = (client) => {
 
   for (const file of eventFiles) {
     const event = require(path.join(eventsPath, file));
-    if (event.once) {
-      client.once(event.name, event.execute);
-    } else {
-      client.on(event.name, event.execute);
-    }
+
+    const eventHandler = (...args) => event.execute(...args, client);
+
+    event.once
+      ? client.once(event.name, eventHandler)
+      : client.on(event.name, eventHandler);
```

Comment:
Ternary operator works better during assignments, when you will not do any assignment is better to use `if-else` which is clearer.

```js
if (event.once) client.once(event.name, eventHandler)
else client.on(event.name, eventHandler);
```